gamemodes/terrortown/gamemode/client/cl_damage_indicator.lua

```diff
@@ -0,0 +1,58 @@
+---
+-- @section Damage Indicator
+
+DMGINDICATOR = {}
+DMGINDICATOR.themes = {}
+
+local dmgindicator_enable = CreateClientConVar("ttt_dmgindicator_enable", "1", true)
+local dmgindicator_mode = CreateClientConVar("ttt_dmgindicator_mode", "Default", true)
+local dmgindicator_duration = CreateClientConVar("ttt_dmgindicator_duration", "1.5", true)
+local dmgindicator_maxdamage = CreateClientConVar("ttt_dmgindicator_maxdamage", "50.0", true)
+local dmgindicator_maxalpha = CreateClientConVar("ttt_dmgindicator_maxalpha", "255", true)
```

Comment:
you could do it here like I've done it in the armor system or like it is done in the karma system and add these cvars to the table.